### PR TITLE
Updated permissioning CLI options and permissioned network tutorial 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated permissioning CLI options and permissioned network tutorial

### CODE REVIEW

File: docs/Getting-Started/Run-Docker-Image.md [hunk 1]
```diff
@@ -144,7 +145,11 @@ Where `mygenesis.json` is your custom configuration file and `path` is the absol
 
 ## Permissions Configuration File 
 
-Specify a permissions configuration file. This is equivalent to specifying the `--permissions-config-file` option.
+Specify a permissions configuration file. This is equivalent to specifying the `--permissions-accounts-config-file`
```
Comment (NicolasMassart) [inline docs/Getting-Started/Run-Docker-Image.md]: you write about two options but to me they are the same `--permissions-accounts-config-file` and `--permissions-accounts-config-file`

File: docs/Reference/Pantheon-CLI-Syntax.md [hunk 2]
```diff
@@ -550,60 +550,123 @@ The default is 30303. Ports must be [exposed appropriately](../Configuring-Panth
...
+    This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#permissions-configuration-file).
+
+### permissions-nodes-contract-address
+
+```bash tab="Syntax"
+--permissions-nodes-contract-address=<ContractAddress>
+```
+
+```bash tab="Example Command Line"
+--permissions-nodes-contract-address=xyz
```
Comment (NicolasMassart) [inline docs/Reference/Pantheon-CLI-Syntax.md]: shouldn't we have a valid address that would also show if we have to use the 0x prefix ?

File: docs/Getting-Started/Run-Docker-Image.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Getting-Started/Run-Docker-Image.md]: Good point - fixed the second node to be nodes.

File: docs/Reference/Pantheon-CLI-Syntax.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/Pantheon-CLI-Syntax.md]: Yes - I'll add this when I'm doc'ing the functionality (not available yet).  Adding a note to the JI so I don't forget.

**Pull request merged**